torch/csrc/throughput_benchmark/ThroughputBenchmark.h

```diff
@@ -0,0 +1,164 @@
+#pragma once
+
+#include <ATen/core/ivalue.h>
+#include <torch/csrc/jit/script/module.h>
+
+#include <vector>
+#include <memory>
+
+namespace torch {
+namespace throughput_benchmark {
+
+/**
+ * The class is used to provide results of a benchmark to the caller
+ * In the future all additional statics should be added here. This class
+ * is exposed as an object to python with the following read-only properties:
+ *
+ * stats.qps
+ * stats.num_forwards
+ * stats.latency_avg_ms
+ *
+ */
+class ExecutionStats {
+public:
+  float latency_avg_ms() const {
```

Comment:
I think pybind should just let you bind fields directly without getters/setters - so this entire thing can be a struct (https://pybind11.readthedocs.io/en/stable/classes.html#instance-and-static-fields - def_readonly, def_readwrite)